src/PrestaShopBundle/Service/DataProvider/Admin/CategoriesProvider.php

```diff
@@ -238,8 +238,7 @@ private function findModuleType(ApiModule $installedProduct, array $modulesTheme
      */
     private function findModuleCategory(ApiModule $installedProduct, array $categories)
     {
-        $moduleCategory = $installedProduct->attributes->get('categoryName');
-        $moduleCategoryParent = $this->getParentCategory($moduleCategory);
+        $moduleCategoryParent = $installedProduct->attributes->get('categoryParentEnglishName');
```

Comment:
That simple?! How come this was a random bug then? (sometimes it's correctly sorted sometimes not at all)
Nothing to do with API response failure?

Reply:
It's because this field has been recently added in addons.prestashop.com :smile_cat: 
For the random, it's not, it working only for en_US, and fr_FR because some category have the same name (Administration, Design & Navigation) for example.

Reply:
Alright!